README.md

```diff
@@ -32,7 +32,15 @@ Where `play-xx` is `play-25`, `play-26` or `play-27` depending on your version o
 
 ## Usage
 
-Examples can be found [here](https://github.com/hmrc/http-verbs/blob/master/http-verbs-common/src/test/scala/uk/gov/hmrc/examples/Examples.scala)
+Examples can be found [here](https://github.com/hmrc/http-verbs/blob/master/http-verbs-common/src/test/scala/uk/gov/hmrc/http/examples/Examples.scala)
+
+### URLs
+
+Urls can be supplied as either java.net.URL objects or as strings (java.net.URL is recommended). A URL interpolator has been implicitly provided with http-verbs for convenience. 
+
+```
+url"http://localhost:20001/create-user"
```

Comment:
Can you include query-params and path-params in the example? This is where the value is.

Reply:
I think we should link to this documentation - https://sttp.softwaremill.com/en/latest/model/uri.html. It has extensive documentation.

Reply:
The only difference is the final type. They build their type `Uri` and we build a `java.net.URL`